Twig/Extension/SonataAdminExtension.php

```diff
@@ -437,6 +437,11 @@ protected function getTemplate(
         try {
             $template = $environment->loadTemplate($templateName);
         } catch (\Twig_Error_Loader $e) {
+            @trigger_error(
+                'Relying on default template loading on field template '.
+                'loading exception is deprecated and will be removed in 4.0',
```

Comment:
I don't get it : 

- The try catch will be removed, which is what I already said.
- There will be no exception to throw.